### PR TITLE
Tooltip rework

### PR DESCRIPTION
**Pull request by YannickMG**
### Several dimension name fixes
 - Handle tooltip display for individual recipes when multiple are visible (current behavior is to  show tooltips for both at the same time)
 - Don't hide the background when tooltip is shown
 - Tooltip trigger area more precise (Used to trigger anytime your mouse was bottom portion of the screen.
 - Dimension names appear in progression order both in abbreviated and tooltip forms
 - Customizable dimension name tooltip max line count (By default, will wrap around at a maximum of 11 lines, creating several columns)
 - Dimension names fixed to match in-game Galacticraft representation, several typos and string display fixes
 - Bumped version up to 1.0.10

### New behavior screenshots

Example of the new tooltip style :

Recipe tooltips handled individually :

Small ores got the same changes :

Dimensions are ordered in progression order :

### CODE REVIEW

File: src/main/java/pers/gwyog/gtneioreplugin/util/GT5CFGHelper.java [hunk 1]
```diff
@@ -154,7 +154,7 @@ public static String GT5CFG(String Veinname) {
 
                 HashSet<String> rawboolsset = new HashSet<String>();
                 if (!rawbools.isEmpty()) {
-                    //remove dublicats
+                    //remove dublicates
```
Comment (bombcar) [inline src/main/java/pers/gwyog/gtneioreplugin/util/GT5CFGHelper.java:157]: should be duplicates, but it's a comment so eh

File: src/main/resources/assets/gtneioreplugin/lang/en_US.lang [hunk 2]
```diff
@@ -37,10 +37,10 @@ gtnop.nei.genPositionInfo=In the center of each chunk
 # world names
 gtnop.world.Overworld=Overworld
 gtnop.world.Nether=Nether
-gtnop.world.TheEnd=TheEnd
-gtnop.world.Twilight=Twilight Forrest
```
Comment (bombcar) [inline src/main/resources/assets/gtneioreplugin/lang/en_US.lang:41]: test this VERY CAREFULLY we have had stuff explode real bad when the Forrest got renamed ... but I can't remember exactly what .... if it's in a lang file maybe ok?

Comment (YannickMG) [reply src/main/resources/assets/gtneioreplugin/lang/en_US.lang:41]: Well, I've consistently been able to boot the full modpack, and I can't imagine anything else depends on the gtnop.world.Twilight lang file entry.

File: src/main/java/pers/gwyog/gtneioreplugin/plugin/gregtech5/PluginGT5Base.java [hunk 3]
```diff
@@ -54,25 +45,51 @@ protected static String getGTOreUnlocalizedName(short index) {
...
+     * 
+     * @param gui An instance of the currentscreen
+     * @param recipe The recipe index being handled
+     * @param dimNames Dimension names to produce a rectangle for
+     * @return Rectangle area of dimension names
+     */
+    protected Rectangle getDimensionNamesRect(GuiRecipe gui, int recipe, String dimNames) {
+        int height = dimNames.length() > 70 ? 30 : (dimNames.length() > 36 ? 20 : 10);        
+        Point offset = gui.getRecipePosition(recipe);
+        return new Rectangle(gui.guiLeft + offset.x + 2, gui.guiTop + offset.y + 110, gui.xSize - 9, height );
```
Comment (YannickMG) [inline src/main/java/pers/gwyog/gtneioreplugin/plugin/gregtech5/PluginGT5Base.java:92]: This is the only place where Access Transformers/Reflection is currently of use. I've got a Reflection commit ready if we don't want to handle this differently.

**Review by bombcar — APPROVED** [review]

**Pull request merged**